services/venice-controller/src/main/java/com/linkedin/venice/controller/systemstore/SystemStoreRepairTask.java

```diff
@@ -378,4 +467,24 @@ Version getNewSystemStoreVersion(String clusterName, String systemStoreName, Str
     return UserSystemStoreLifeCycleHelper
         .materializeSystemStore(getParentAdmin(), clusterName, systemStoreName, pushJobId);
   }
+
+  int getRepairJobCheckTimeoutInSeconds() {
```

Comment:
do we really need this getters? can use the statics directly?

Reply:
This is for unit test to override to second level. By default it is sub-hour level which make it not testable for unit test